test/box/tuple.result

```diff
@@ -364,6 +364,34 @@ t[nil]
 - null
 ...
 --------------------------------------------------------------------------------
+-- test tuple:format
+--------------------------------------------------------------------------------
+f = box.tuple.format.new({{'id', 'number'}, {'name', 'string'}})
+---
+...
+t = box.tuple.new({1, 'Flint', true}, {format = f})
+---
+...
+t:format()
+---
+- [{'name': 'id', 'type': 'number'}, {'name': 'name', 'type': 'string'}]
+...
+box.tuple.format(t)
+---
+- [{'name': 'id', 'type': 'number'}, {'name': 'name', 'type': 'string'}]
+...
+t = box.tuple.new({2, 'a', false})
+---
+...
+t:format()
+---
+- []
```

Comment:
Maybe it should return a nil in this case.

Reply:
But space:format() without a specified format returns an empty format object. So it could be an expected behavior.